proposals/meshery.md

```diff
@@ -1,85 +1,129 @@
-== Meshery Proposal
+Meshery CNCF Proposal
 
-*Name of project:* Meshery
+Name of project: Meshery
 
-*Description:* Meshery is an open source, multi-service mesh management plane.
+Description: Meshery is an open source multi-service mesh management plane.
 
-=== Alignment with CNCF - Why does CNCF need a multi-service mesh management plane?
+Alignment with CNCF - Why does CNCF need a multi-service mesh management plane?
+===============================================================================
```

Comment:
Can we remove this line, it's making the above line too huge?
<img width="1066" alt="Screenshot 2020-04-06 at 1 47 00 AM" src="https://user-images.githubusercontent.com/44091822/78509007-94a0d200-77a8-11ea-928c-3581fb2af3a1.png">